test/code-fixtures/projects/ts/.eslintrc.json

```diff
@@ -15,5 +15,6 @@
 	"ignorePatterns": [
 		"lib/**",
 		"node_modules/**"
-	]
+	],
+	"root": true
```

Comment:
Question: Is this the desired behavior? Do we want the custom configuration to cascade into whatever configuration is in the targeted folder, or do we want it to be used *instead* of the one in the target folder? Those aren't equivalent, so we should decide.
[Here](https://forcedotcom.github.io/sfdx-scanner/en/custom-config/eslint/#users-responsibilities) is what our documentation has to say on the matter. It seems to imply that it should cascade, but it's a little bit unclear.
We can make it entirely ignore the existing config by also setting `noEslintrc = true` in the config object we use to instantiate the engine, if that's the behavior we decide we want.

Reply:
Here is my take - let me know your thoughts:

Eslint's natural style seems to be [cascading config unless specifically disabled](https://eslint.org/docs/user-guide/configuring/configuration-files#cascading-and-hierarchy) (with the `root=true` you've added).

Since we are expecting eslint's NPM dependencies to be set by users if they plan to customize config, allowing eslint to work the way it does without our influence seems to be the natural way of working.

Having `root=true` in our test cases seems like a good idea so that the tests don't have additional influence, but I'm not sure if a user would necessarily look at it the same way.